### ISSUE TITLE
Edit post notification content

### ISSUE THREAD

**Issue by joshuaalpuerto** [OneSignal/react-native-onesignal#253]
Is there a way I can actually add `subtitle`, `header` on this code?

```
// Calling postNotification
let data = arr // some array as payload
let contents = {
	'en': 'You got notification from user'
}
OneSignal.postNotification(contents, data, playerId);
```

I tried going through the [docs](https://documentation.onesignal.com/reference#create-notification) but I can't get it working.

Thanks

**Comment by neo125874** [comment]
@joshuaalpuerto hi, did you figure this out? 
could you share your array or object, i don't know about the completely look it should be..thanks!

**Comment by joshuaalpuerto** [comment]
@neo125874, actually the same as the documentation.

```
OneSignal.postNotification({
   'en': 'Someone is calling.'
}, {}, user.push_id)
```

The problem is when im adding `subtitle` or `header` notification won't send.

**Comment by neo125874** [comment]
@joshuaalpuerto thanks for your info, i found that the 'en' must be in, even when other language code was setting.

one more question, would badge or title(not subtitle) works?

**Comment by joshuaalpuerto** [comment]
unfortunately I haven't tried using those properties.

**Comment by neo125874** [comment]
@joshuaalpuerto so the alert info window, you custom that? or how did you solve the title & content would be the same?

**Comment by joshuaalpuerto** [comment]
@neo125874 , customization didn't work. Im stuck to the default settings

```
OneSignal.postNotification({
   'en': 'Someone is calling.'
}, {}, user.push_id)
```

**Comment by leoskyrocker** [comment]
Hi @joshuaalpuerto @neo125874, extra options can be passed into the function like this:
```OneSignal.postNotification(contents, data, playerId, {subtitle: 'subtitle', headings: {"en": "English Title", "es": "Spanish Title"}})```

It is undocumented and implemented since v3.0.5 by https://github.com/geektimecoil/react-native-onesignal/pull/266
